taskManager.js

```diff
@@ -0,0 +1,93 @@
+const fs = require('fs');
+
+const TASKS_FILE = 'tasks.json'; // File where tasks are stored
+
+function loadTasks() {
```

Comment:
Rewrite all functions into lambda function

Reply:
Since these functions will not be longer declared as named functions, the console.log statements will now print the messages to the console of the lambda function's environment instead of the standard output, so I dont think I have to change them